contracts/binding/src/lib.rs

```diff
@@ -2,25 +2,33 @@
 #[macro_use]
 mod convert;
 #[allow(clippy::all)]
-pub mod i_diamond;
+pub mod checkpointing_facet;
 #[allow(clippy::all)]
-pub mod diamond_loupe_facet;
+pub mod cross_msg_helper;
```

Comment:
@cryptoAtwill something on your end still formats this code into alphabetical ordering. Is that because you manually open and save this file, or is there something else at play? Could it be something that explains why we couldn't get a deterministic build?

Reply:
what I did was `make compile-abi`, `make rust-binding` and `cargo fmt`. Is it because the contract changed that's causing this?

Reply:
Aha, probably the `cargo fmt` part does it. It's not done by `make` at any point (my editor does it on save), and the generated source is deliberately [ignored](https://github.com/consensus-shipyard/ipc/blob/main/rustfmt.toml#L4) during format checks. 

Yes if you have any change in the contracts then you will get a completely fresh `lib.rs`; on my machine it's left as-is, the modules are in the order of appearance in `build.rs`. So when I build stuff, it will undo your changes, and because I don't use `cargo fmt`, we'll flip between ordered and unordered modules.